### ISSUE TITLE
Content-Length is 0 for NamedFile HEAD request

### ISSUE THREAD

**Issue by iBelieve** [actix/actix-web#914]
According to https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests, the proper way to implement resumable downloads is to issue a HEAD request, check for the `Accept-Ranges` header, and then issue a GET request with the `Range` header set.

The problem is that Actix-web is setting `Content-Length` to 0 for HEAD responses, so I'm not able to issue a range request to the end of the file:

```
$ http --headers GET ...
HTTP/1.1 200 OK
accept-ranges: bytes
content-disposition: attachment; filename="..."
content-length: 9513784
content-type: ...
date: Wed, 12 Jun 2019 16:21:57 GM
```

```
http HEAD ...
HTTP/1.1 200 OK
accept-ranges: bytes
content-disposition: attachment; filename="..."
content-length: 0
content-type: ...
date: Wed, 12 Jun 2019 16:28:56 GMT
```

In my server, I am using Actix-web 1.0 and am returning a `NamedFile`:

```
NamedFile::open(path)?
    .use_etag(false)
    .use_last_modified(false)
```

I've also tried specifically setting `Content-Length` directly, and it still doesn't work. 

I haven't looked too deeply into the actix-web source code yet, but it looks like the content-length might be getting overridden: https://github.com/actix/actix-web/blob/9bb40c249fc18fbeb2b72e012aa3b411893a7fa5/actix-http/src/client/h2proto.rs#L58

**Comment by fafhrd91** [comment]
fixed in 0.1.2

**Comment by iBelieve** [comment]
Great, thank you for the quick fix. I can confirm it's working as expected in my project.
